Content.Server/Destructible/Thresholds/Triggers/DamageGroupTrigger.cs

```diff
@@ -11,13 +13,14 @@ namespace Content.Server.Destructible.Thresholds.Triggers
     /// </summary>
     [Serializable]
     [DataDefinition]
-    public class DamageClassTrigger : IThresholdTrigger
+    public class DamageGroupTrigger : IThresholdTrigger
     {
-        /// <summary>
-        ///     The class to check the damage of.
-        /// </summary>
-        [DataField("class")]
-        public DamageGroupPrototype? Group { get; set; }
+
+        // TODO QUESTION Why is damage type allowd to be null here? if it is, this component does nothing? Surely it should be a required datafield?
+        // TODO PROTOTYPE Replace this datafield variable with prototype references, once they are supported.
+        [DataField("damageGroup")]
```

Comment:
Is there a use for ThresholdTrigger with a null damage group?

Reply:
should be equired imo